leveller.js

```diff
@@ -61,16 +61,24 @@
     var diff = targetHeight - currentHeight;
     if (diff === 0) return;
 
+    var isHeightProperty = (this.options.cssProperty.indexOf('eight') > 0);
```

Comment:
No love for `this.options.cssProperty.toLowerCase().indexOf('height') !== -1` ? Eh, it's all good.